python/ray/dashboard/consts.py

```diff
@@ -22,10 +22,12 @@
 # Max allowed number of in-progress requests could be configured.
 RAY_STATE_SERVER_MAX_HTTP_REQUEST_ALLOWED = 1000
 
-RETRY_REDIS_CONNECTION_TIMES = 10
-CONNECT_REDIS_INTERNAL_SECONDS = 2
-PURGE_DATA_INTERVAL_SECONDS = 60 * 10
-ORGANIZE_DATA_INTERVAL_SECONDS = 2
+RAY_DASHBOARD_STATS_PURGING_INTERVAL = env_integer(
+    "RAY_DASHBOARD_STATS_PURGING_INTERVAL", 60 * 10
+)
+RAY_DASHBOARD_STATS_UPDATING_INTERVAL = env_integer(
+    "RAY_DASHBOARD_STATS_UPDATING_INTERVAL", 2
+)
```

Comment:
Let's also make all updates in Node head configurable

Reply:
done